android/src/main/java/com/opentokreactnative/OTSessionManager.java

```diff
@@ -598,7 +598,6 @@ public void onStreamReceived(Session session, Stream stream) {
         ConcurrentHashMap<String, Boolean> destroyedStreams = sharedState.getPublisherDestroyedStreams();
         boolean streamWasDestroyed = destroyedStreams.containsKey(streamId);
         if (streamWasDestroyed) {
-            destroyedStreams.remove(streamId);
```

Comment:
I'm not removing the fix.  This change keeps the **streamIds** in the map because there's a case where we can receive the destroyed stream (as a new stream) more than 1 time. Actually on iOS is already like this

https://github.com/timurgaitov/opentok-react-native/blob/416318bd38c26b8dd7f7105bfcc4a9a863e5132c/ios/OpenTokReactNative/OTSessionManager.swift#L485-L487